### PR TITLE
Make Login & Sign Up as modal layovers

### PR DESCRIPTION
**Pull request by yining1023**
This PR changes Login and Sign Up from separate pages to modal layovers on the main page.

screen shot 2016-10-05 at 3 09 10 pm
screen shot 2016-10-05 at 3 09 29 pm

### CODE REVIEW

File: client/routes.js [hunk 1]
```diff
@@ -3,8 +3,8 @@ import React from 'react';
 import App from './modules/App/App';
 import IDEView from './modules/IDE/pages/IDEView';
 import FullView from './modules/IDE/pages/FullView';
-import LoginView from './modules/User/pages/LoginView';
-import SignupView from './modules/User/pages/SignupView';
+// import LoginView from './modules/User/pages/LoginView';
```
Comment (catarak) [inline client/routes.js]: No need to leave this in commented out, just delete it!

File: client/styles/components/_login.scss [hunk 2]
```diff
@@ -1,9 +1,11 @@
 .login {
+	@extend %modal;
 	text-align: center;
-	height: 100%;
+	// height: 100%;
```
Comment (catarak) [inline client/styles/components/_login.scss]: No need to leave this in commented out.

File: client/styles/components/_signup.scss [hunk 3]
```diff
@@ -1,9 +1,11 @@
 .signup {
+	@extend %modal;
 	text-align: center;
-	height: 100%;
+	// height: 100%;
```
Comment (catarak) [inline client/styles/components/_signup.scss]: No need to leave this in commented out.

File: client/routes.js (hunk 1 above)
Comment (catarak) [inline client/routes.js]: No need to leave this in commented out, I would just remove it.

File: client/styles/components/_login.scss (hunk 2 above)
Comment (catarak) [inline client/styles/components/_login.scss]: No need to leave this in commented out, I would just remove it.

File: client/styles/components/_signup.scss (hunk 3 above)
Comment (catarak) [inline client/styles/components/_signup.scss]: No need to leave this in commented out, I would just remove it.

**Comment by catarak** [comment]
Overall this looks great 😄 and I'm not seeing that React error you were having.

**Comment by yining1023** [comment]
😄 Yay! I just deleted all extra comments, and committed the changes.

**Pull request merged**